<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>683</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Ident FS</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="projectList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>241</width>
      <height>451</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="resizeMode">
     <enum>QListView::ResizeMode::Adjust</enum>
    </property>
    <property name="spacing">
     <number>5</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAddFiles">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dateien modden</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDescription1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Klicken sie auf ein Projekt um mehr Informationen zu erhalten und &lt;span style=&quot; font-weight:700;&quot;&gt;Dateien&lt;/span&gt; hinzuzufügen.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="blank1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="resizeMode">
     <enum>QListView::ResizeMode::Adjust</enum>
    </property>
    <property name="spacing">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditProjectName">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>60</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::StrongFocus</enum>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Projektname</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCreateProject">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Projekt erstellen</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupNewProject">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>130</y>
      <width>538</width>
      <height>385</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="labelCreated">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>109</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Projekt erstellt: </string>
     </property>
    </widget>
    <widget class="QLabel" name="labelProjectName">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>222</width>
       <height>53</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Projektname</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelDescription2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>69</y>
       <width>518</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Sie können registrierte Dateien zu Projekten hinzufügen. Eine Datei kann in mehreren Projekten verwendet werden.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::TextFormat::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>1</number>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelLastChange">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>131</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Projekt geändert:</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidgetFilesInProject">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>153</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidgetFiles">
     <property name="geometry">
      <rect>
       <x>272</x>
       <y>153</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonCancel">
     <property name="geometry">
      <rect>
       <x>447</x>
       <y>351</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonOk">
     <property name="geometry">
      <rect>
       <x>359</x>
       <y>351</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxDescription">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>130</y>
      <width>601</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="labelProjectName_3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>441</width>
       <height>53</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Willkommen bei identFS</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelDescription1_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>541</width>
       <height>261</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;IdentFS ist ein Werkzeug zur Verwaltung und Überwachung von Dateien mit Fokus auf Integrität, Nachvollziehbarkeit und Struktur.&lt;br/&gt;Das System speichert zu jeder relevanten Datei eindeutige Identifikationsdaten wie Hashwerte, Metadaten und Pfadinformationen, um Änderungen zuverlässig erkennen zu können.&lt;/p&gt;&lt;p&gt;Ein Projekt in IdentFS repräsentiert einen logisch abgegrenzten Dateibestand.&lt;br/&gt;Jedes Projekt besitzt eine eigene UUID, einen definierten Speicherort (Vault) und eine zugehörige Datenbank. Innerhalb eines Projekts werden Dateien einmalig identifiziert und dauerhaft überwacht.&lt;/p&gt;&lt;p&gt;IdentFS ist darauf ausgelegt, auch große Datenmengen effizient zu verwalten. Statt Dateien permanent neu zu scannen, nutzt das System inkrementelle Verfahren und Dateisystem-Ereignisse, um Änderungen gezielt zu erfassen. Dadurch bleibt die Anwendung performant, selbst bei sehr großen Volumes.&lt;/p&gt;&lt;p&gt;Ziel von IdentFS ist es, Transparenz über Dateizustände zu schaffen, Manipulationen erkennbar zu machen und eine verlässliche Grundlage für Archivierung, Vergleich und Versionierung zu bieten.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <zorder>blank1</zorder>
   <zorder>projectList</zorder>
   <zorder>pushButtonAddFiles</zorder>
   <zorder>labelDescription1</zorder>
   <zorder>lineEditProjectName</zorder>
   <zorder>pushButtonCreateProject</zorder>
   <zorder>groupNewProject</zorder>
   <zorder>groupBoxDescription</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
